### PR TITLE
Adding two utility functions for use in the shell course.

### PR DESCRIPTION
**Pull request by gvwilson**
1. `test_condition(state, boolean, msg)` reports an error if the boolean condition isn't met.
2. `test_cwd(state, expected, msg)` reports an error if the user is not currently in the expected directory.

### CODE REVIEW

File: shellwhat_ext/__init__.py [hunk 1]
```diff
@@ -3,8 +3,38 @@
...
+def test_cwd(state, expected, msg=None):
+    '''Check whether the user is currently in the expected directory.
+    This is a convenience wrapper around a rather obscure shell check:
+    see https://github.com/datacamp/learn-bugs/issues/62 for the reason
+    that the shell expression is needed.
+    '''
+
+    if msg is None:
+        msg = 'Not in expected directory "{}"'.format(expected)
+    test_expr_error(state, "[[ $PWD == '{}' ]]".format(expected), msg=msg)
```
Comment (filipsch) [inline shellwhat_ext/__init__.py]: I'm not convinced about this one. Can you add some unit tests that would show it's used?

**Comment by filipsch** [comment]
I'd love to see some unit tests before merging this in.

**Comment by gvwilson** [comment]
Yup - still experimenting with the API as I work through the shell lesson chapter.

**Comment by gvwilson** [comment]
Could someone please merge or close.

**Pull request merged**